### PR TITLE
docs: add PWA cookbook

### PR DESCRIPTION
**Pull request by userquin**

### CODE REVIEW

File: PWA.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Elk will generate 2 web manifests per locale, one for light theme and one for dark theme.
+
+You can check web manifest generation on `modules/pwa/i18n.ts` module.
+
+## PWA UI Components
+
+
+
+## Debugging PWA in development
```
Comment (edimitchel) [inline PWA.md:67]:
```suggestion
### Debugging PWA in development
```

File: PWA.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- Tap the "Share" icon at the bottom of the screen.
+- Scroll down and tap "Add to Home Screen".
+- Customize the name of the app (if desired) and tap "Add".
+- The PWA should now appear on your home screen.
+
+### Firefox
+
+To install a Progressive Web App (PWA) on Firefox, follow these steps:
+- Tap the menu button (three horizontal lines) at the bottom of the screen.
+- Tap "Page options" and then "Install website as app".
```
Comment (edimitchel) [inline PWA.md:37]: No Page options on my Android firefox, directly the Install action button:

File: PWA.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- Customize the name of the app (if desired) and tap "Add".
```
Comment (edimitchel) [inline PWA.md:38]: On android I don't have the possibility to update name

Comment (userquin) [reply PWA.md]: too lazy, using ChatGPT, seems it is outdated 😅

**Review by edimitchel — APPROVED** [review]

**Comment by userquin** [comment]
@patak-dev we can merge it, we can later add/modify some entries

**Comment by patak-dev** [comment]
Is this a page in the docs? Shouldn't it be linked there?

**Comment by userquin** [comment]
moving page to docs...

File: README.md [hunk 4]
```diff
@@ -117,6 +117,9 @@ Elk uses [Vitest](https://vitest.dev). You can run the test suite with:
 nr test
 ```
 
+## 📲 PWA
+
+You can consult the [PWA documentation](https://docs.elk.zone/docs/pwa) to learn more about the PWA capabilities on Elk, how to install Elk PWA in your desktop or mobile device and some hints about PWA stuff on Elk.
```
Comment (edimitchel) [inline README.md:122]:
```suggestion
You can consult the [PWA documentation](https://docs.elk.zone/docs/pwa) to learn more about the PWA capabilities on Elk, how to install Elk PWA in your desktop or mobile device and some hints about PWA stuff on Elk.

```

**Review by edimitchel — APPROVED** [review]

**Comment by patak-dev** [comment]
@danielroe is it possible to use Docus without the order in the file name? I see that @userquin modified a filename here to change it and I'm worrying about the git history later on if we need to reorder pages. What do you think?

**Comment by danielroe** [comment]
I _think_ it's just needed if you want to override the order - so you can have multiple files with same number without a problem.

**Comment by userquin** [comment]
> @danielroe is it possible to use Docus without the order in the file name? I see that @userquin modified a filename here to change it and I'm worrying about the git history later on if we need to reorder pages. What do you think?

I can remove the numbering, but the PWA entry will appear after privacy...

**Comment by danielroe** [comment]
maybe make it more like `80` so it's not so close to it in order?

**Comment by userquin** [comment]
I want to put privacy last on the nav bar, that's why I use 99, changing pwa entry to 80...

**Pull request merged**